zkvm: fix path to cmath in zkvm module

marijanp opened the pull request:
I don't know why the original author decided to use relative paths.

I think it would be better to use `use crate::sys::cmath;`

The according issue can be found here https://github.com/risc0/risc0/issues/1647

joboet commented:
`unix::cmath` doesn't exist anymore. You can just remove the `mod`, `cmath` has been moved to `sys` and is shared by all platforms (see #120109).

marijanp committed:
zkvm: remove cmath

- Remove cmath from zkvm module since cmath was moved to sys and is
shared by all platforms (see #120109)

joboet commented:
Cool!

@bors r+ rollup

bors commented:
:pushpin: Commit 861e213f878d1e0412a6eb57ba33025adf047bee has been approved by `joboet`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=joboet 861e213f878d1e0412a6eb57ba33025adf047bee -->
<!-- homu: {"type":"Approved","sha":"861e213f878d1e0412a6eb57ba33025adf047bee","approver":"joboet","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->
